front/src/store/modules/tickets.js

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+const { serverPath } = require('@/config/config.dev.json');
+
+export default {
+  namespaced: true,
+  state: {
+    tickets: []
+  },
+  mutations: {
+    SET_USER_TICKETS (state, tickets) {
+      state.tickets = tickets;
+    }
+  },
+  actions: {
+    refreshTickets ({ commit }) {
+      axios
+        .get(`${serverPath}/api/tickets`)
+        .then(({ data }) => {
+          commit('SET_USER_TICKETS', data);
+        });
+    }
+  },
+  getters: {
+    tickets (state) {
+      return state.tickets;
```

Comment:
NTH: use getters for data manipulation not to just take data state. see example in docs https://vuex.vuejs.org/ru/guide/getters.html
so, this getter is unnecessary 

NTH - **N**ice **T**o **H**ave -  It would make the code better by changing/fixing this. Review will pass.